core/cap-0035.md

```diff
@@ -552,6 +553,8 @@ Possible return values for the `SetTrustLineFlagsOp` during application are:
 - `SET_TRUST_LINE_FLAGS_CANT_REVOKE` if `AUTH_REVOCABLE_FLAG` is not set on the issuer,
   but the authorization is downgraded (`AUTHORIZED_FLAG` -> `AUTHORIZED_TO_MAINTAIN_LIABILITIES_FLAG`, `AUTHORIZED_FLAG` -> 0,
   or `AUTHORIZED_TO_MAINTAIN_LIABILITIES_FLAG` -> 0).
+- `SET_TRUST_LINE_FLAGS_INVALID_STATE` if the final state of the trustline flag has both
+  `AUTHORIZED_FLAG` and `AUTHORIZED_TO_MAINTAIN_LIABILITIES_FLAG` set.

```

Comment:
❗ One paragraph up this situation is described as what triggers the malformed error code. I think the malformed error code needs including into this list and updating.

Reply:
That scenario is different than this one - The malformed case checks the `setFlags` parameter during validation, while this one checks the actual flag during application.